src/Cli/dotnet/NugetPackageDownloader/LocalizableStrings.resx

```diff
@@ -138,4 +138,7 @@
   <data name="FailedToValidatePackageSigning" xml:space="preserve">
     <value>Failed to validate package signing.</value>
   </data>
+  <data name="FailedToGetPackageUnderPackageSourceMapping" xml:space="preserve">
+    <value>Package Source Mapping is enabled but no feeds matched the specified packed ID: {0}. See the documentation for Package Source Mapping at aka.ms/nuget-package-source-mapping for more details.</value>
```

Comment:
```suggestion
    <value>Package Source Mapping is enabled but no feeds matched the specified packed ID: {0}. See the documentation for Package Source Mapping at https://aka.ms/nuget-package-source-mapping for more details.</value>
```

Reply:
Also a nit (on mobile at the moment): the phrase 'packed ID' should probably be 'package ID', right?

Reply:
Also, a missing comma before the "but"

Reply:
And maybe lock the URL in the localization comments. Vendors should know not to translate, but locked strings are supposed to fail validation tests on the vendor side.